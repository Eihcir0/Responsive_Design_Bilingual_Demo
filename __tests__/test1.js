import React from 'react';
import NavBarLeft from './../components/nav_bar_left';
import renderer from 'react-test-renderer';

test('button switches language', () => {

  const componentDK = renderer.create(
    <NavBarLeft language="DK"/>
  );
  let treeDK = componentDK.toJSON();
  expect(treeDK).toMatchSnapshot();
});

test('button switches language', () => {
  const componentEN = renderer.create(
    <NavBarLeft language="EN"/>
  );
  let treeEN = componentEN.toJSON();
  expect(treeEN).toMatchSnapshot();
});

// future direction for testing:
// - Test valid email entered
// - Test API posts for email
// - Test page switching
// - Test login
