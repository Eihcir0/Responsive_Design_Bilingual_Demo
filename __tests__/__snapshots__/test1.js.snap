exports[`test button switches language 1`] = `
<section
  className="nav-bar">
  DK
  <div
    className="nav-bar-left-side">
    <button
      className="nav-bar-button"
      onClick={undefined}>
      TIL FORBRUGEREN
    </button>
    <button
      className="nav-bar-button"
      onClick={undefined}>
      TIL VIRKSOMHEDEN
    </button>
    <button
      className="nav-bar-button"
      onClick={undefined}>
      OM ROGER
    </button>
  </div>
</section>
`;

exports[`test button switches language 2`] = `
<section
  className="nav-bar">
  EN
  <div
    className="nav-bar-left-side">
    <button
      className="nav-bar-button"
      onClick={undefined}>
      FOR CONSUMERS
    </button>
    <button
      className="nav-bar-button"
      onClick={undefined}>
      FOR business
    </button>
    <button
      className="nav-bar-button"
      onClick={undefined}>
      ABOUT US
    </button>
  </div>
</section>
`;
